src/powerquery-language-services/inspection/activeNode/activeNode.ts

```diff
@@ -32,6 +32,8 @@ export interface ActiveNode extends IActiveNode {
     readonly ancestry: ReadonlyArray<TXorNode>;
     // A conditional indirection to the leaf if it's an Ast identifier.
     readonly maybeIdentifierUnderPosition: Ast.Identifier | Ast.GeneralizedIdentifier | undefined;
+    // A conditional indirection to the leaf if it's an Ast identifier of an included range.
```

Comment:
It's unclear to me the difference between the two by name only. Both include mention of a position which implies a range for both of them.

Reply:
I wanted to pick it up in the case of |foo where `|` is the cursor position.